osh/hub/scan/models.py

```diff
@@ -1283,3 +1283,26 @@ def csmock_args(self):
             # there are no arguments, this is not an error
             logger.info("No csmock arguments for profile '%s'", self)
             return ''
+
+
+class RetentionPolicySettingsManager(models.Manager):
+    def get_days_from_category(self, category):
+
+        # We establish a broad default days, so we don't delete unwanted results.z
+        try:
+            days = self.get(category=category).days
+        except ObjectDoesNotExist:
+            logger.warning("Category %s could not be found. Setting default parameter", category)
+            days = 60
+
+        return days
+
+
+class RetentionPolicySetting(models.Model):
+    category = models.CharField(max_length=128, blank=False, null=False)
+    days = models.PositiveIntegerField(
+        default=14,
+        help_text="Number of days before the text is deleted "
```

Comment:
Sorry, missed this one in the previous round.  What `"text"` is the field description referring to?  I believe you meant logs, task artifacts, etc.  Also, the description string contains a trailing white-space character.